server/db/models/review.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var _ = require('lodash');
+var Sequelize = require('sequelize');
+
+module.exports = function (db) {
+
+    db.define('review', {
+      rating: {
+        type: Sequelize.INTEGER,
+        validate: {
+          min: 0,
+          max: 5,
+          notNull: true
+        }
+      },
+      description: {
```

Comment:
Reviews need to be at least a certain amount of characters (per the workshop).  Add a validate with a len validate of [40]?